firstvoices/backend/tests/test_apis/test_widgets_api.py

```diff
@@ -167,6 +164,20 @@ def test_list_minimal(self):
         # Skipping test as it is same as the test_list_permissions test above.
         pass
 
+    @pytest.mark.skip(
+        reason="Site widget API does not have eligible optional charfields."
```

Comment:
The only optional charfields on widgets is the "value" in the key/values pairs within the widget settings list, so a null value would just be [] I believe